app/global.css

```
@@ -3,13 +3,16 @@
-    --secondary: #FF8C42;
+    --secondary: #414B5A;
     --text: #E5E5E5;
     --text-title: #FFFFFF;
     --card-hover-px: 20px;
     --card-border-radius: 5px;
-
+    --mobile-font-size: 1.1em;
+    --backdrop-blur: 50px;
     box-sizing: border-box;
+
+    interpolate-size: allow-keywords;
```

Biome (1.9.4)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)